modules/mvt/src/lib/binary-vector-tile/features-to-binary.js

```diff
@@ -0,0 +1,314 @@
+// Convert binary features to flat binary arrays
+// @ts-nocheck
+
+export function featuresToBinary(features, firstPassData, options = {}) {
```

Comment:
Can we add JSDoc header to each function and remove the ts-nocheck?

Small time investment that will help review and maintenance.

Reply:
Done

Reply:
@ibgreen OK so while I added the header, I cannot figure out what to do regarding the Typescript. If I remove the `ts- nocheck` directive then I get some errors (basically this issue https://stackoverflow.com/questions/52423842/what-is-not-assignable-to-parameter-of-type-never-error-in-typescript) and I don't see how I can fix them without converting the whole file to typescript (admittedly, I'm no Typescript expert). But I don't see any other place in the codebase that does this. Could you give some guidance on how to proceed?